src/ext/tags.scss

```diff
@@ -151,19 +151,31 @@
     }
 
     /* Used for grouping tags together */
-    &.group-tags .tag {
-        margin-right: 0 !important;
-
-        &:first-child {
-            border-radius: 0.25rem 0 0 0.25rem;
-        }
-    
-        &:not(:first-child):not(:last-child) {
-            border-radius: 0;
+    &.group-tags {
+        /* Round first and last tag in the grouped tags */
+        &.group-rounded {
```

Comment:
To try to stick to BEM naming conventions moving forward, we should name this `group-tags--rounded`.